specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/standardAssignments.json

```diff
@@ -0,0 +1,373 @@
+              "type": "string"
+            }
+          }
+        },
+        "metadata": {
+          "$ref": "#/definitions/StandardAssignmentMetadata"
+        }
+      }
+    },
+    "StandardAssignmentMetadata": {
+      "type": "object",
+      "description": "The standard assignment metadata",
+      "properties": {
+        "createdBy": {
+          "description": "standard assignment Created by object id (GUID)",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdOn": {
+          "description": "standard assignment creation date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "updatedBy": {
+          "description": "standard assignment last updated by object id (GUID)",
+          "type": "string",
+          "readOnly": true
+        },
+        "updatedOn": {
+          "description": "standard assignment last update date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "AssignedStandardItem": {
+      "type": "object",
+      "description": "describe the properties of a of a standard assignments object reference",
+      "properties": {
+        "id": {
+          "description": "full resourceId of the Microsoft.Security/standard object",
+          "type": "string"
+        }
+      }
+    },
+    "AssignedAssessmentItem": {
+      "type": "object",
+      "description": "describe the properties of a security assessment object reference (by key)",
+      "properties": {
+        "assessmentKey": {
+          "description": "unique key to a security assessment object",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "StandardAssignmentName": {
+      "name": "standardAssignmentName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "pattern": "[{]?[0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$",
+      "description": "The standard assignments assignment key - unique key for the standard assignment",
+      "x-ms-parameter-location": "method"
+    },
+    "StandardsAssignmentBody": {
+      "name": "standardAssignment",
+      "in": "body",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/StandardAssignment"
+      },
+      "description": "Custom standard assignment over a pre-defined scope",
+      "x-ms-parameter-location": "method"
+    },
+    "Scope": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The scope of the standard assignment. Valid scopes are: management group (format: 'providers/Microsoft.Management/managementGroups/{managementGroup}'), subscription (format: 'subscriptions/{subscriptionId}'), or security connector (format: 'subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/securityConnectors/{securityConnectorName})'",
```

Comment:
>security connector (format: 'subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/securityConnectors/{securityConnectorName})' [](http://example.com/codeflow?start=234&length=168)

Same here, for using security connector as scope.

Reply:
Not sure why this is a problem, a proxy resource can be extension to another resource. In this case the customRecommendations is an extension to the security connector.
I see live examples in the manifest for this behavior in governancerules.